src/main/java/com/imsweb/x12/Separators.java

```diff
@@ -22,6 +23,12 @@ public Separators() {
         setSegment('~');
         setElement('*');
         setCompositeElement(':');
+        if (System.getProperty("os.name").toLowerCase().contains("win")) {
```

Comment:
I think it would be better if we could try to detect the line break from the file itself when parsing the file initially.

Reply:
you cannot detect CRLF vs LF using a scanner, unless I'm mistaken. So this would cause us to need to do a step prior to parsing that detects line feeds. Any objection to this?

Reply:
Maybe something like this:
1. Set the default to be no line breaks. I have seen a lot of X12 files without any line breaks in them so this seems OK to do.
2. Have two toX12String methods, one that takes a LineBreak as a parameter and one that doesn't.
3. Have the method that takes the LineBreak parameter set the line break if it is not null.

Reply:
Yep i was thinking the same. will do this.